clients/stellar-relay-lib/src/connection/services.rs

```diff
@@ -298,10 +305,12 @@ pub(crate) async fn connection_handler(
 				_connection_handler(action, &mut connector, &mut w_stream).await?;
 			},
 
-			Ok(None) => {},
+			Ok(None) => {
+				log::info!("why is it none?");
+			},
 
 			Err(elapsed) => {
-				log::error!("{} for receiving messages.", elapsed.to_string());
+				log::info!("{} for receiving messages.", elapsed.to_string());
```

Comment:
This message probably also is not accurate (?). If I understand correctly it should be something like "Connection timed out after {} seconds."